asf_core_data/pipeline/preprocessing/feature_engineering.py

```diff
@@ -379,7 +375,7 @@ def get_postcode_coordinates(df, postcode_field_name="POSTCODE"):
     df = data_cleaning.reformat_postcode(df, white_space="remove")
 
     # Merge with location data
-    df = pd.merge(df, postcode_coordinates_df, on=postcode_field_name, how="outer")
+    df = pd.merge(df, postcode_coordinates_df, on=postcode_field_name, how="left")
```

Comment:
This function isn't referenced in any pipeline but is similar functionality with asf_core_data.pipeline.mcs.process.process_mcs_utils.
geocode_postcode()

Could you clarify which you'd intend to keep?

Reply:
good catch @sqr00t ! I am actually not sure why this was created, as I am more familiar with the MCS pipeline than EPC - @athrado where do you use this function?

Reply:
Right, good catch, I meant to use it at the very end after merging all datasets, but the MCS data already has the long/lat loaded earlier in the pipeline so I ended up forgetting to add it for EPC... 

Maybe you can quickly double-check whether the EPC only records (no MCS match) really don't have the long/lat. If so, we can add a line in merging_pipeline() right after loading epc. 

`feature_engineering.get_postcode_coordinates(prep_epc, postcode_field_name="POSTCODE")`. That should do it!

Reply:
We could also add it to the EPC processing pipeline but I found it sometimes takes a bit long and isn't needed for every use case so I removed as a default processing step.

Reply:
hey @athrado , thanks for clarifying!

- I have added the missing line of code to `merging_pipeline()` so that we have the coordinates for EPC records;
- @sqr00t I guess to answer your question, for asf_daps you can add only one of the functions to the utils (either the one in `feature_engineering.py` or the one in `process_mcs_utils.py`), since they do roughly the same - the code just needs a bit of adapting;